projects/plugins/protect/src/js/state/actions.js

```diff
@@ -103,6 +138,85 @@ const ignoreThreat = ( threatId, callback = () => {} ) => async ( { dispatch } )
 	} );
 };
 
+const getFixThreatsStatus = threatIds => async ( { dispatch } ) => {
+	return await apiFetch( {
+		path: `jetpack-protect/v1/fix-threats?threat_ids=${ threatIds }`,
```

Comment:
In our backend we only accept POST requests to this endpoint which will trigger a POST request to wpcom. We should add a new GET endpoint that will trigger a GET to wpcom so this call won't cause an 404.

Reply:
Done!